artiq/remoting.py

```diff
@@ -59,7 +59,11 @@ def run_command(self, cmd, **kws):
 
 class SSHClient(Client):
     def __init__(self, host, jump_host=None):
-        self.host = host
+        if host.find("@") >= 0:
```

Comment:
``@`` in ``host``

Are you sure Paramiko doesn't already support parsing those strings somehow?

Reply:
I copied this solution from paramiko demo, so I suspect no. Also username is a separate argument in `connect`.

https://github.com/paramiko/paramiko/blob/master/demos/demo_simple.py#L56